app/Models/Count.php

```diff
@@ -36,4 +38,11 @@ public static function totalUsers()
     {
         return static::find('usercount')->count ?? 0;
     }
+
+    public static function cachedTotalUsers()
+    {
+        return Cache::remember('count_totalUsers', 5, function () {
```

Comment:
makes it a redis lookup instead of a db lookup, figured it would be worthwhile seeing as this is used on the top page and stuff  ┐('～`；)┌